support/awsutil/sg.go

```diff
@@ -148,5 +148,16 @@ func DefaultWorkerSGIngressRules(vpcCIDRBlock, sgGroupID, sgUserID string) []*ec
 				},
 			},
 		},
+		{
```

Comment:
2 comments:
This overlaps https://github.com/openshift/hypershift/pull/2475/files#diff-30470382ea59639d6dd3a0f2e763d8fbdf5d7961f1f70329d63f7b7068519de5R33-R42 so we should at least remove that.

For the private link endpoint, I think all the ports we use are 6443 and 443. Would it make sense to try only opening those?